src/main/java/com/fasterxml/jackson/databind/cfg/CacheProvider.java

```diff
@@ -11,5 +16,11 @@
 public interface CacheProvider
     extends java.io.Serializable
 {
-    // !!! TODO: add methods
+    /**
+     * Method to provide a {@link LookupCache} instance for {@link DeserializerCache}
+     * 
+     * @return {@link LookupCache} instance for caching {@link JsonDeserializer}s
+     */
+    LookupCache<JavaType, JsonDeserializer<Object>> provideForDeserializerCache();
```

Comment:
could rename simply as `forDeserializerCache`. And if possible, pass `DeserializationConfig` as argument.

Reply:
N/m defined in 4103

Reply:
Yes, I missed that before adding comment, realized right after.